### ISSUE TITLE
Unknown error: GPG is unavailable on this device

### ISSUE THREAD

**Issue by hirschnase** [keybase/client#3337]
After a fresh install of the OS X client app I signed in and got this error message. GPG is installed as commandline utility... Not sure what the problem is here...

my log id: 05845a6848b9365d63f9e51c

![screen shot 2016-07-05 at 19 22 16](https://cloud.githubusercontent.com/assets/163801/16593554/20fb62c0-42e6-11e6-9389-3de506c6dd5e.png)

**Comment by cjb** [comment]
Hi!  We've fixed this, but the new build containing the fix isn't released yet.  I'll comment here once it is, should be very soon.

**Comment by hirschnase** [comment]
Cool, thanks.

**Comment by hirschnase** [comment]
Btw is there a workaround this that I can apply? Would be cool if I could start using the app :)

**Comment by MLWALK3R** [comment]
So has this been release yet because im getting this issue.

**Comment by cjb** [comment]
@MLWALK3R Yes, it's been released now -- please could you run `keybase log send` so we can check it out?

**Comment by MLWALK3R** [comment]
https://github.com/keybase/client/issues/4732
